civo/provider.go

```diff
@@ -92,19 +100,16 @@ func providerConfigure(d *schema.ResourceData) (interface{}, error) {
 		return nil, fmt.Errorf("[ERR] token not found")
 	}
 
-	var client *civogo.Client
-	var err error
-
-	apiURL, envExists := os.LookupEnv("CIVO_API_URL")
-	if envExists && apiURL != "" {
-		client, err = civogo.NewClientWithURL(tokenValue, apiURL, regionValue)
-		if err != nil {
-			return nil, err
-		}
-		log.Printf("[DEBUG] Civo API URL: %s\n", apiURL)
-		return client, nil
+	if api_url, ok := d.GetOk("api_url"); ok {
+		apiURL = api_url.(string)
+	} else {
+		apiURL = ProdAPI
 	}
-
+	client, err = civogo.NewClientWithURL(tokenValue, apiURL, regionValue)
+	if err != nil {
+		return nil, err
+	}
+	log.Printf("[DEBUG] Civo API URL: %s\n", apiURL)
```

Comment:
Remove this line, we have the same line at the end of the file

Reply:
Done